packages/appeals-service-api/src/lib/notify.js

```diff
@@ -78,7 +78,28 @@ async function sendEmail(appeal) {
     logger.debug({ options }, 'options');
     await notifyClient.sendEmail(templateId, destinationEmail, options);
   } catch (err) {
-    logger.error({ err }, 'Problem sending email');
+    const message = 'Problem sending email';
+    if (err.response) {
+      logger.error(
+        {
+          message: err.message,
+          data: err.response.data,
+          status: err.response.status,
+          headers: err.response.headers,
+        },
```

Comment:
any use to spreading the rest of `err` in the `if` / `else if` flows?

Reply:
Possibly. It's the same as in the all the Horizon functions so I have confidence that this works - I don't know what else is in that `response` object that shouldn't be in the logs